### DISCUSSION TITLE
Add support for listing quoting posts (of a given post)

### DISCUSSION THREAD

**Discussion by spaghetti-source** [bluesky-social/atproto#1401] [Bluesky Lexicons]
In Twitter, users can see the QTs (quote tweets) of a tweet. This feature is quite useful to read other users' opinion/comments/etc about the tweet. Can we have the same feature in Bluesky?

**Comment by chocorayto** [comment]
I was using Twitter same way, reading the quote RTs to know peoples' opinions on certain subjects - it would enable further discussions too, which will make bluesky more interesting :-)
